src/System.Resources.Reader/src/System.Resources.Reader.csproj

```diff
@@ -6,19 +6,21 @@
     <AssemblyName>System.Resources.Reader</AssemblyName>
     <RootNamespace>System.Resources</RootNamespace>
     <ProjectGuid>{16EE5522-F387-4C9E-9EF2-B5134B043F37}</ProjectGuid>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
   </PropertyGroup>
   <!-- Default configurations to help VS understand the configurations -->
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="System\Resources\ResourceReader.cs" />
-    <Compile Include="System\Resources\StreamExtensions.cs" />
+    <Compile Include="../ref/System.Resources.Reader.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../System.Resources.ResourceManager/ref/System.Resources.ResourceManager.csproj"/>
```

Comment:
I don't understand why are we doing this.

---
In reply to: [82480996](https://github.com/dotnet/corefx/pull/12468#discussion_r82480996) [](ancestors = 82480996)